src/components/__tests__/markdownPreview.test.js

```diff
@@ -0,0 +1,447 @@
+import React from "react";
+import renderer from "react-test-renderer";
```

Comment:
Cool!
Could you change the testfile to use `enzyme-to-json` library and get rid of `react-test-renderer`  - there is no trace of it in `master` now. How to work with it you can see [here](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L26)

Reply:
It was clearly mentioned in the issue to use "react-test-renderer" which is why I used it.

Reply:
I have updated the test case and used the enzyme library instead. The coverage report can be seen in the image below.

![image](https://user-images.githubusercontent.com/31980558/97113244-d5434700-170a-11eb-84ad-719d05147dbe.png)